packages/next/server/hot-reloader.js

```diff
@@ -173,8 +173,9 @@ export default class HotReloader {
     await this.clean()
 
     this.wsPort = await new Promise((resolve, reject) => {
+      const { devWebSocketPort } = this.config
       // create dynamic entries WebSocket
-      this.wss = new WebSocket.Server({ port: 0 }, function (err) {
+      this.wss = new WebSocket.Server({ port: devWebSocketPort || 0 }, function (err) {
```

Comment:
Let's always read from `devWebsocketPort` and add `0` as a default in packages/next-server/server/config.js` Then we have a single source of truth 👍

Reply:
Good idea, I updated it